journal-of-urban-and-environemntal-engineering.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Journal of Urban and Environmental Engineering</title>
+    <id>http://www.zotero.org/styles/journal-of-urban-and-environemntal-engineering</id>
```

Comment:
You have a typo in the file name, style ID and "self" link: "environemntal" should be "environmental" (switch the "em" to "me")